modules/storages/app/controllers/storages/admin/storages_controller.rb

```diff
@@ -85,7 +85,7 @@ def select_provider
     service_result = ::Storages::Storages::SetAttributesService
                  .new(user: current_user,
                       model: @object,
-                      contract_class: Storages::Storages::BaseContract,
+                      contract_class: EmptyContract,
                       contract_options: { skip_provider_type_strategy: true })
```

Comment:
_nit:_ We  no longer need to [skip_provider_type_strategy](https://github.com/opf/openproject/blob/dev/modules/storages/app/contracts/storages/storages/base_contract.rb#L51) if we've switched from `Storages::Storages::BaseContract`
```suggestion
                      contract_class: EmptyContract)
```

Reply:
removed